### ISSUE TITLE
Unable to load SDK-style projects that lack `Sdk` attribute

### ISSUE THREAD

**Issue by sandyarmstrong** [mono/monodevelop#5712]
The following project loads fine in VS 15.7.4, but fails to load in VSmac 7.6 build 2086:

```xml
<!-- Notice no Sdk="Microsoft.NET.Sdk". This prevents the project from loading
     in VSmac, though it loads fine in VSwin. Avoiding the SDK attribute and
     manually doing SDK props/targets importing is often suggested as a
     workaround for bugs like https://github.com/NuGet/Home/issues/5894. -->
<Project>
    <!-- This is implicit with Project@Sdk="Microsoft.NET.Sdk" -->
    <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />

    <PropertyGroup>
        <TargetFramework>net472</TargetFramework>
    </PropertyGroup>

    <!-- This is implicit with Project@Sdk="Microsoft.NET.Sdk" -->
    <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
</Project>
```

The following error appears:

![image](https://user-images.githubusercontent.com/155076/44281554-0cee0480-a20d-11e8-94e4-06bff8b23414.png)

In the log:

```
ERROR [2018-08-17 10:41:57Z]: Error while trying to load the project /Users/sandy/xam-git/simple/simple.csproj
MonoDevelop.Projects.MSBuild.UnknownSolutionItemTypeException: Unknown solution item type
  at MonoDevelop.Projects.ProjectService+<>c__DisplayClass25_0.<ReadSolutionItem>b__0 () [0x000ca] in /Users/vsts/agent/2.136.1/work/1/s/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectService.cs:125 
  at MonoDevelop.Core.Runtime+<>c__42`1[T].<RunInMainThread>b__42_0 (System.Object state) [0x00046] in /Users/vsts/agent/2.136.1/work/1/s/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Core/Runtime.cs:393 
```

**Comment by mrward** [comment]
This is a known limitation - explicit imports are not as yet supported - https://bugzilla.xamarin.com/show_bug.cgi?id=58564 (VSTS 522315)

There is a [pull request](https://github.com/mono/monodevelop/pull/5597) with a fix that I need to review/test.

**Comment by mrward** [comment]
This should now be fixed.

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.8.0.150 (master)

Pull Request #5713 merged by: Matt Ward
Author: mono
Commit: 9666995b8c795bf14725f69776364d173b41f8c1 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.7.0.1070 (release-7.7)

Pull Request #5713 merged by: Matt Ward
Author: mono
Commit: 9666995b8c795bf14725f69776364d173b41f8c1 (mono/monodevelop)
